e2e/wdio/platform/tests/time-picker.e2e-spec.ts

```diff
@@ -0,0 +1,130 @@
+import {
+    checkElementScreenshot,
+    click,
+    doesItExist,
+    getAttributeByName,
+    getElementArrayLength, getImageTagBrowserPlatform,
+    getText,
+    getValue,
+    refreshPage, saveElementScreenshot,
+    scrollIntoView,
+    sendKeys,
+    setValue,
+    waitForElDisplayed,
+    waitForPresent
+} from '../../driver/wdio';
+import { time, text, defaultValidTime } from '../fixtures/testData/time-picker';
+import { TimePickerPO } from '../pages/time-picker.po';
+
+describe('Time picker suite', function() {
+    let timePickerPage: TimePickerPO;
+    timePickerPage = new TimePickerPO();
```

Comment:
is it needed to make timePickerPage type TimePickerPO?
maybe just const timePickerPage = new TimePickerPO(); is enough?